crypto/src/main/java/org/hyperledger/besu/crypto/Signature.java

```diff
@@ -0,0 +1,136 @@
+package org.hyperledger.besu.crypto;
+
+import com.google.common.base.Suppliers;
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.bytes.MutableBytes;
+import org.apache.tuweni.units.bigints.UInt256;
+
+import java.math.BigInteger;
+import java.util.Objects;
+import java.util.function.Supplier;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class Signature {
+
+    public static final int BYTES_REQUIRED = 65;
```

Comment:
Again, we shouldn't lock in the signature length.  It is reasonable to prepare for chains that may require stronger EC cures.

Reply:
I guess the impact of supporting stronger keys might add extra work that won't provide much value atm. One of the assumptions for this is that secp256k1 and secp256r1 use keys of the same size. If we make the size dynamic, we might need to handle this across different layers of the system.

I'm not saying that we shouldn't do it, but maybe not in this first iteration.

Reply:
We can expose the key length as a signature API and the initial batch of factory methods can presume it.  But I'm wary of baking in 65 bytes too deeply into the API.